### PR TITLE
APP-174: [C++] Basic level Apps: Credentials management

### PR DESCRIPTION
**Pull request by RostakaGmfun**
/cc @slash32 @Dubland

### CODE REVIEW

**Comment by Oldnickus** [comment]
@kaa-leeroy test this please

**Comment by Oldnickus** [comment]
@kaa-leeroy test this please

File: credentialsdemo/source/cpp/build.sh [hunk 1]
```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
```
Comment (balldir) [inline credentialsdemo/source/cpp/build.sh]: sh is preferred https://github.com/kaaproject/kaa/pull/940

Comment (balldir) [inline credentialsdemo/source/cpp/build.sh]: Overall LGTM

**Comment by vadimol** [comment]
why Java code is also included in this PR?
Push it in separate PR (and probably a separate Jira) or add reviewers for Java code.

**Comment by vadimol** [comment]
@kaa-leeroy test this please

**Comment by Oldnickus** [comment]
@kaa-leeroy retest this please

**Comment by Slash32** [comment]
@kaa-leeroy retest this please

**Comment by Slash32** [comment]
@kaa-leeroy retest this please

**Comment by Slash32** [comment]
@kaa-leeroy retest this please

**Comment by Slash32** [comment]
LGTM

**Pull request merged**